.vsts-ci.yml

```diff
@@ -43,34 +43,41 @@ stages:
           queue: BuildPool.Windows.10.Amd64.VS2019.Pre.Open
         ${{ if ne(variables['System.TeamProject'], 'public') }}:
           name: NetCoreInternal-Pool
-          queue: buildpool.windows.10.amd64.vs2017
```

Comment:
Do we need the changes in this file?

Reply:
yes. This is not something arcade can automatically handle